rundocker.sh

```diff
@@ -1,13 +1,7 @@
-# NOTE: kolibri whl file must be located in the current directory
-
-mkdir -p bin
-rm bin/*.apk 2> /dev/null
-
-docker build -t kolibrikivy .
+#!/bin/bash
 
 docker run -i --rm -v ${PWD}/bin:/mnt/bin kolibrikivy /bin/bash << COMMANDS
```

Comment:
Was trying to move this to the Makefile but I don't think the `COMMANDS` was being recognized